src/components/home-page/home-page.jsx

```diff
@@ -19,6 +20,7 @@ const HomePage = () => (
     <div>
         <EmptyBlock>Upcoming events</EmptyBlock>
         <EmptyBlock>Passed events</EmptyBlock>
```

Comment:
I depends on where I should add the title, maybe it would be better to replace it on the block with title?